pyani_plus/private_cli.py

```diff
@@ -775,6 +775,7 @@ def anim(  # noqa: C901, PLR0912, PLR0913, PLR0915
         # Try to abort gracefully without wasting the work done.
         msg = f"Interrupted, will attempt to log {len(db_entries)} completed comparisons\n"
         sys.stderr.write(msg)
+        run.status = "Worker interrupted"
```

Comment:
I think we ought to define some constants or even an ENUM for assorted status strings?

See also ``run.status = "Running"`` special casing in #245.